servant-foreign/src/Servant/Foreign/Internal.hs

```diff
@@ -7,7 +7,7 @@
 -- arbitrary programming languages.
 module Servant.Foreign.Internal where
 
-import           Control.Lens hiding (cons, List)
+import           Control.Lens (makePrisms, makeLenses, Getter, (&), (<>~), (.~), (%~))
```

Comment:
FWIW, i dislike this kind of imports with lens. There are just a few name clashes and you have to import something if you have to use a slightly different operator/combinator.